glib/Marshaller.cs

```diff
@@ -88,15 +88,19 @@ static string Utf8PtrToStringFast (IntPtr ptr, int len)
 			}
 		}
 
+#if HAVE_NET_4_6
```

Comment:
If you're doing a check at buildtime then you don't need a reflection check at runtime.